source/common/upstream/load_balancer_impl.h

```diff
@@ -44,18 +44,20 @@ class LoadBalancerBase {
   Runtime::Loader& runtime_;
   Runtime::RandomGenerator& random_;
 
-  // TODO(alyssawilk) make load balancers priority-aware and remove.
-protected:
-  const HostSet& host_set_;
-
 private:
   enum class LocalityRoutingState { NoLocalityRouting, LocalityDirect, LocalityResidual };
 
+  /**
+   * Increase per_priority_state_ to at leat |size|
```

Comment:
Nit: s/leat/least/. Also, remove blank line. I think it would be helpful to state the invariant that per_priority_state_ must obey, i.e. the motivation for resizing above.